frontend/src/components/uam/common/types/users-table-actions-props.type.ts

```diff
@@ -1,6 +1,7 @@
 type UsersTableActionsProps = {
   id: number;
   onDelete: (id: number) => void;
+  currentLoggedInUser: boolean;
```

Comment:
Don't forget that boolean vars supposed to be named as an answer to general question ([B7](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md))
eg
```suggestion
  isCurrentSignedUser: boolean;
```